ui/src/app/auth/auth-roles.ts

```diff
@@ -1,5 +1,15 @@
 // Default values of OSMT Roles
-export const OSMT_ADMIN = "ROLE_Osmt_Admin"
-export const OSMT_CURATOR = "ROLE_Osmt_Curator"
+const OSMT_ADMIN = "ROLE_Osmt_Admin"
```

Comment:
I would eventually like to provide the values for these role keys via an env.js, the same way we would any other runtime/config values. This is good for now. @hikr17 , please confirm?

Reply:
env.js is for lane-specific values.  I would suggest that ROLES are not lane-specific, but specific to an adopter of the tech.  So, I would disagree on putting the roles in env.js.